newm-server/src/main/kotlin/io/newm/server/database/migration/V10__SongsUpdates.kt

```diff
@@ -0,0 +1,14 @@
+package io.newm.server.database.migration
```

Comment:
Why 2 separate migrations: V9 and V10?

Reply:
Just because I had already run the first migration in my test environments. It was easier to do the new updates in a new migration.